server/components/auth/index.js

```diff
@@ -126,6 +126,11 @@
           res.locals.isSystemAdministrator = isSystemAdministrator(req, res);
           res.locals.isTeamLeader = isTeamLeader(req, res);
 
+          // Hold onto a court location id for places that need one. Potentially a problem!
+          if (res.locals.isCourtUser) {
+            req.session.locCode = req.session.authentication.locCode;
+          }
+
```

Comment:
Not sure on any implications of this.
MIght be one to double check with @ricardofreitasrocha

Reply:
Yep, I'm nervous!

Reply:
it should get reset in places where we actually need `req.session.locCode` but in most of those places that is set also from `req.session.authentication.locCode`. fingers crossed nothing breaks 😄